recipe/build.sh

```diff
@@ -1,17 +1,23 @@
-#!/bin/bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./build-aux
+#!/usr/bin/env bash
 
+set -o xtrace -o nounset -o pipefail -o errexit
+
+export CFLAGS="${CFLAGS} -Wno-implicit-function-declaration"
+
+if [[ ${target_platform} == "linux-aarch64" || ${target_platform} == "linux-ppc64le" ]]; then
+    sed -i '?tests/time-max-rss.sh \\?d' Makefile.am
```

Comment:
```suggestion
    sed -i '/tests/time-max-rss.sh \\/d' Makefile.am
```

Reply:
I guess this also needs the `/` after `tests` to be escaped.  Let's see if the next commit fixes it.